Makefile

```diff
@@ -0,0 +1,13 @@
+
+
+REQUIRED_BINS := xsel dmenu clipnotify
```

Comment:
dmenu isn't required if you're going to use a different `CM_LAUNCHER`.

Reply:
`dmenu` is a hard dependency on Arch Linux. I was modeling what's done in the PKGBUILD file: https://git.archlinux.org/svntogit/community.git/tree/trunk/PKGBUILD?h=packages/clipmenu

By depending on `dmenu`, you have a better chance of success for a very common case.

Like a lot of `rofi` users, I symlink `dmenu` to rofi so that every `dmenu` call is redirected. Even a number of `rofi` users will have `dmenu` in their path. That's the only way to handle some other `dmenu` utilities.